frontend/packages/dev-console/locales/en/devconsole.json

```diff
@@ -362,12 +362,13 @@
   "Role": "Role",
   "Select a role": "Select a role",
   "Reload": "Reload",
-  "<0>{children} or <3>create a Project.</3></0>": "<0>{children} or <3>create a Project.</3></0>",
+  "or": "or",
+  "create a Project.": "create a Project.",
```

Comment:
We aren't able to break up sentences like this. It's problematic for translation. We should find a way to make this a single message (using the `Trans` component if we need to add a button).

Reply:
Thanks @spadgett , have updated PTAL